.github/workflows/jit_triage.yml

```diff
@@ -0,0 +1,72 @@
+name: jit-triage
+
+on:
+  issues:
+    types: [opened]
```

Comment:
Issue labels are often added after an issue has been opened (either by the pytorch bot or by a human triager). We should cover that case as well.

Reply:
I added `labeled` to the types field, which should handle this. However, it will not cover the case in which the JIT label is removed from an issue and needs to be removed from the project board. Should we do that too?

Maybe later once we get this working.

Reply:
yeah later is fine